### PR TITLE
Game state machine

### PR DESCRIPTION
**Pull request by MyrrhDev**
Integrating game state logic machine

### CODE REVIEW

File: camera.js [hunk 1]
```diff
@@ -2,8 +2,7 @@ import * as posenet_module from '@tensorflow-models/posenet';
 import * as facemesh_module from '@tensorflow-models/facemesh';
 import * as tf from '@tensorflow/tfjs';
 import * as paper from 'paper';
-import dat from 'dat.gui';
-
+import Stats from 'stats.js';
```
Comment (sachsom95) [inline camera.js]: is stat.js required?

File: camera.js [hunk 2]
```diff
@@ -152,7 +152,6 @@ function brainLoaded() {
 }
 
 function classifyPose() {
-  // console.log('Classify pose');
```
Comment (sachsom95) [inline camera.js]: probably keep all the console logs for now

File: camera.js [hunk 3]
```diff
@@ -189,9 +186,41 @@ function modelLoaded() {
   console.log('poseNet ready');
   var splash = document.getElementById("loader");
   splash.classList.add('display-none');
+}
 
+
```
Comment (sachsom95) [inline camera.js:191]: make this a separate module and export the function

File: camera.js (hunk 1 above)
Comment (MyrrhDev) [reply camera.js]: true, it's not

File: index.html [hunk 4]
```diff
@@ -10,16 +10,14 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
     <link href="poseman.css" rel="stylesheet">
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
-    <script src="/scripts/p5.js"></script> 
-    <!-- <script src="/scripts/p5.sound.min.js"></script> -->
```
Comment (sachsom95) [inline index.html:14]: is p5 sound library being used? or is it for future use. Maybe include it when we work on sound

File: camera.js (hunk 2 above)
Comment (MyrrhDev) [reply camera.js]: done

File: package.json [hunk 5]
```diff
@@ -17,10 +17,11 @@
     "face-api.js": "^0.22.1",
     "paper": "^0.12.1",
     "parcel-plugin-static-files-copy": "^2.5.1",
-    "stats.js": "^0.17.0"
```
Comment (sachsom95) [inline package.json:20]: is stats.js required?

**Review by sachsom95 — COMMENTED** [review]
do checkout the nits

File: camera.js (hunk 3 above)
Comment (MyrrhDev) [reply camera.js:191]: Need clarification for "this", unless you just mean the gotPoses() function

File: index.html (hunk 4 above)
Comment (MyrrhDev) [reply index.html:14]: I guess not, so I'm happy to comment it out

Comment (MyrrhDev) [reply index.html:14]: Done

File: package.json (hunk 5 above)
Comment (MyrrhDev) [reply package.json:20]: removed

File: camera.js (hunk 3 above)
Comment (sachsom95) [reply camera.js:191]: The game Play function gameOn and startCountdown

**Review by MyrrhDev — COMMENTED** [review]
> do checkout the nits

The... what?

**Review by sachsom95 — COMMENTED** [review]
yes this is perfect

**Pull request merged**